AppController/djinn.rb

```diff
@@ -5144,11 +5122,7 @@ def initialize_scaling_info_for_app(app_name, force=false)
     @current_req_rate[app_name] = 0
     @total_req_rate[app_name] = 0
     @last_sampling_time[app_name] = Time.now.to_i
-
-    if !@last_decision.has_key?(app_name)
-      @last_decision[app_name] = 0
-    end
-
+    @last_decision[app_name] = 0 unless @last_decision.has_key?(app_name)
```

Comment:
I know that this works, but the way it reads is ambiguous. I could see it as either
`(@last_decision[app_name] = 0) unless @last_decision.has_key?(app_name)`
or
``@last_decision[app_name] = (0 unless @last_decision.has_key?(app_name))`